wvd-templates/DSC/Functions.ps1

```diff
@@ -62,12 +62,12 @@ class PsRdsSessionHost {
         if (($operation -eq "get") -and $this.CheckForAvailableState) {
             $StartTime = Get-Date
 
-            while ($sessionHost.Status -ine "Available") {
+            while ($sessionHost.Status -ine "Available" -and $sessionHost.Status -ine "NeedsAssistance") {

```

Comment:
do you think it would be better to check if `$sessionHost.Status -ieq "Unavailable"` instead ? In case they add other states ?

Reply:
It seems safer to have a specific list in this case since if new statuses are added in the future they could also mean that something closer to Unavailable than Available.